packages/docs/page-config/ui-elements/slider/examples/Slots.vue

```diff
@@ -11,7 +12,8 @@
   <va-slider v-model="value2">
     <template #append>
       <va-input
-        v-model.number="value2"
+        v-model="value2Proxy"
+        @input="e => e.target.value = value2Proxy"
```

Comment:
Maybe we should do it inside VaInput. I feel like it is not obvious to user (and for the one who wrote this doc example too). I'd expect v-model to be the same as input.value. What do you think?

Reply:
Hi, @m0ksem, I agree with you, I will try to put it inside VaInput.

Reply:
Hi, @m0ksem, I think I can put the logic in `onInput` function in the `useCleave` hook, but I am not sure that is a good idea, but I can't find a way to put into the VaInput.vue. Can you give me some pointers?

Reply:
I think you can manually add `@input` event to handle it separately from useCleave.